### PR TITLE
[Dynamo][Benchmarks] Add Qwen3.5-0.8B to Hugging Face benchmarks

### PR DESCRIPTION
**Pull request by kevzhuo**
### Background
Add Qwen3.5 coverage to the Dynamo text-generation benchmarks.

The pinned Transformers 5.13.0 supports loading the text component of `Qwen/Qwen3.5-0.8B` and its hybrid attention cache, so the model can use the existing `TextGenerationBenchmark` adapter.

### What changed

1. **Model registration** — register `Qwen/Qwen3.5-0.8B` with the existing LLM batch-size convention (hint 8, divisor 8, effective batch size 1), following the existing CPU skip policy.
2. **Expected results** — add ten CUDA/ROCm rows covering ordinary/dynamic inference, ordinary/dynamic training, and AOTInductor. Existing training and AOT failures are recorded as `eager_fail_to_run` and `fail_to_run` respectively.
3. **Flaky list** — add Qwen3.5 to the existing ROCm LLM graph-metric flaky list.

The model reuses the existing loading, generation, and timing paths. Inference expectations start at `pass,0`; CUDA capture/fallback metrics remain blank pending measured CI results.

### Test plan

Completed locally in the CPU development environment:

| Check | Result |
| --- | --- |
| BF16 Inductor accuracy, standard 1,000-token prompt | Pass — 1 graph, 21,933 captured calls, 0 graph breaks, 0 eager fallbacks | | Eager static-cache generation, 1,000-token prompt + 4 forced output tokens | Pass (shortened compiled-generation perf run also completed) | | 4 suite-selection checks | Pass |
| 10 new-row checks against both CSV validators | Pass (recorded local results used as fixtures) | | `python test/dynamo/test_benchmark_runner.py -v` | Pass (both tests) | | `spin quicklint` | Pass |

> **Note:** Translation validation was disabled locally because z3 is unavailable.
>

### Known caveat

Generation diagnostics found different greedy tokens for closely ranked BF16 eager/compiled logits. Prefill logits passed the existing FP64-based accuracy criterion and warmed output/cache behavior stabilized, but **exact eager/compiled token equality is not established.**

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @kadeng @chauhang @amjames @jataylo @azahed98

### CODE REVIEW

File: benchmarks/dynamo/ci_expected_accuracy/inductor_huggingface_inference.csv [hunk 1]
```diff
@@ -86,6 +86,10 @@ Qwen/Qwen3-0.6B,pass,0,2282,0
 
 
 
+Qwen/Qwen3.5-0.8B,pass,0,,
```
Comment (azahed98) [inline benchmarks/dynamo/ci_expected_accuracy/inductor_huggingface_inference.csv]: Are these empty because you're waiting on numbers from CI?

Comment (azahed98) [reply benchmarks/dynamo/ci_expected_accuracy/inductor_huggingface_inference.csv]: Looks like it is also failing CI with `eager_fail_to_run` https://github.com/pytorch/pytorch/actions/runs/35774580333/job/107369903277?pr=198261#step:12:1570

Comment (kevzhuo) [reply benchmarks/dynamo/ci_expected_accuracy/inductor_huggingface_inference.csv]: Should be fixed in the recent run, it seems like the issue was the missing offline model cache.
